mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/mapping/annotation/Parameter.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.bridge.mapping.annotation;
```

Comment:
My opinion is that if maybe it is better to move it only when we need to share it with other packages.

Reply:
And break API? I'm sorry, but that's not a great plan.

It's an annotation, not an interface that we can alter later to make it extend its replacement in another package. If we put it here and intend to reuse it at some point, it will have to stay here. And given the package name, it will be awkward.